lib/globber.js

```diff
@@ -1,30 +1,36 @@
 'use strict';
-const glob = require('glob');
+const Glob = require('glob').Glob;
 const Rx = require('rx');
+const fs = require('fs');
 
 function globber(paths) {
   const files = new Rx.Subject();
   files.fileEvents = [];
 
-  const Glob = glob.Glob;
   let doneCount = 0;
 
-  paths.forEach(function (path) {
+  function globFiles(path) {
+    try {
+      if (fs.statSync(path).isDirectory()) {
+        path += '/**/*.js';
+      }
+    } catch(e) {}
```

Comment:
this try catch was necessary to avoid abruptions where the path is not expanded by default. This is perceived within the collateral tests. I'm not exactly sure how to make it safe before `fs.statSync(path)`. Any known best practice?